spec/javascripts/Policies/components/HeaderButton.spec.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React from 'react'
+import { mount } from 'enzyme'
+
+import { HeaderButton } from 'Policies/components/HeaderButton'
+
+it('should render itself', () => {
+  const wrapper = mount(<HeaderButton type='add' onClick={jest.fn()} />)
+  expect(wrapper.find(HeaderButton).exists()).toBe(true)
+})
+
+it('should handle clicks', () => {
+  const spy = jest.fn()
+  const wrapper = mount(<HeaderButton type='add' onClick={spy} />)
+
+  wrapper.props().onClick()
+  expect(spy).toHaveBeenCalled()
+})
+
+it('should have different stules for each type', () => {
```

Comment:
Typo: "styles"
Or maybe better to rephrase like: "should render different icons"